### DISCUSSION TITLE
How do I set a multi choice column?

### DISCUSSION THREAD

**Discussion by navarrorc** [MarkMpn/Sql4Cds#370] [Q&A] [answered]
Hi. Please review the images provided below. I'm able to see that the column values are separated by a coma for the Value and semicolon for the Label, how can a create an Update statement that will set this column with multiple values e.g. 0,1,2
It would be so much faster using this tool than having to do it in the Dataverse web editor.

![select_statement](https://github.com/MarkMpn/Sql4Cds/assets/215220/760f57e5-b269-49fe-98cd-da209872bf61)
![choices](https://github.com/MarkMpn/Sql4Cds/assets/215220/22f256ea-6a83-4055-9e21-0b8557f5525d)

**Comment by MarkMpn** [comment, marked as answer]
You should be able to do this like:

```sql
UPDATE systemuser
SET    new_approver_region = '0,1,2'
WHERE  …
```

**Reply by navarrorc** [reply]
Thank you Mark. That worked perfectly.
